torch/_inductor/codegen/triton_utils.py

```diff
@@ -33,6 +38,12 @@ def signature_of(arg: KernelArgType, *, size_dtype: str) -> str:
                 return "fp32"
             else:
                 return new_tye
+        elif zero_dim_cpu_tensor_list and (arg.buffer in zero_dim_cpu_tensor_list):
+            new_tye = tye.lstrip("*")
+            if new_tye.startswith("fp"):
+                return "fp64"
+            else:
+                return "int"
```

Comment:
wrong. will fix it in next commit.

Reply:
fixed in https://github.com/pytorch/pytorch/pull/132702/commits/edc43caba6cf8a0f9aa962e513cac14f0d392aef